tests/integrations/test_notification.py

```diff
@@ -0,0 +1,71 @@
+import boto3
+import json
+from flask import Flask
+
+from vulnerable_people_form.integrations.notifications import send_message
+from vulnerable_people_form.form_pages.shared.session import form_answers
+
+_current_app = Flask(__name__)
+_current_app.secret_key = "test_secret"
+_current_app.is_tiering_logic_enabled = False
+
+
+def test_send_message(sqs):
+    sqs = boto3.client("sqs")
+    queue_url = sqs.list_queues()["QueueUrls"][0]
+
+    with _current_app.test_request_context() as test_request_ctx:
+        _current_app.config["AWS_SQS_QUEUE_URL"] = queue_url
+        test_request_ctx.session["form_answers"] = {
+            "nhs_number": "1234567890",
+            "date_of_birth": {
+                "day": "01",
+                "month": "03",
+                "year": "1994"
+            },
+            "name": {
+                "first_name": "FirstName",
+                "last_name": "LastName",
+            },
+            "contact_details": {
+                "email": "email@test.com"
+            },
+            "support_address": {
+                "building_and_street_line_1": "101 Test Lane",
+                "town_city": "London",
+                "postcode": "SW1A 9AJ"
+            },
+            "nhs_letter": 1,
+            "basic_care_needs": 1
+        }
+
+        submission_id = "test-registration-number"
+        send_message(submission_id)
+
+        sqs_response = sqs.receive_message(QueueUrl=queue_url)
+        message = json.loads(sqs_response["Messages"][0]["Body"])
+
+        print(f'a: {message}')
+
+        postcode = form_answers()["support_address"]["postcode"]
+        if len(postcode) == 6 and " " not in postcode:
```

Comment:
Why is this logic needed? If the postcode is set to SW1A 9AJ I can't see why it would ever be 6

Reply:
@sh31 it feels like we are testing the sqs client, we would just want to test the fact that send message behaves as expected

Reply:
I've seen this logic before in `session.py` - it feels like an early version of postcode handling which doesn't actually match up to the actual postcode system but doesn't actually break anything - we may want to pull both

Reply:
@thomasridd the test has been updated since this version